lib/platform/gitlab/index.md

```diff
@@ -3,3 +3,12 @@
 ## Features awaiting implementation
 
 - The `automergeStrategy` configuration option has not been implemented for this platform, and all values behave as if the value `auto` was used. Renovate will accept the Merge Request without further configuration, and respect the strategy defined in the Merge Request, and this cannot be overridden yet
+
+## Server version dependent features
+
+We use the GitLab [version API](https://docs.gitlab.com/ee/api/version.html) to fetch the server version.
+This can be overridden by the experimental feature flag [`RENOVATE_X_SERVER_VERSION`](https://docs.renovatebot.com/self-hosted-experimental/#renovate_x_server_version).
+
+- Use `Draft:` MR prefix instead of `WIP:` prefix since `v13.2.0`
+- Do not truncate Markdown body to 25K chars since `v13.4.0`
+- Allow configure reviewers since `v13.9.0`
```

Comment:
This list is a bit hard to understand, what do those versions mean? I think we mean to say that "certain features are only available after certain GitLab" versions? But this is not really explained properly in the current wording.

I think we should have a new sentence here that explains what those versions mean.

Reply:
Yes, that are the gitlab server versions, when renovate will change the behavior

Reply:
I'm also not sure, because this is complicated stuff... 😄 

Maybe @rarkins can propose a draft text that we can improve on together?